components/Header/Header.js

```diff
@@ -0,0 +1,19 @@
+import styles from './header.module.scss'
+import Link from 'next/dist/client/link'
+
+const Header = () => {
+  return (
+    <header className="container__dark">
+        <div className='header__container'>
+            <nav className={styles.navbar_content}>
```

Comment:
`styles.header__navbar` ?

The navebar isnt in a separate component so I think it should still be associated with the component block 'header'